types/base.d.ts

```diff
@@ -120,5 +120,13 @@ export interface ContextualCallbackTestContext extends CallbackTestContext {
 	context: any;
 }
 
+export interface Macro<I, E, T> {
```

Comment:
Actually I would put the `T` argument at the first position. Doesn't it make more sense? Because the macro also provides the testcontext as the first argument.

```ts
export interface<T, I, E>
```

Same for `Macros`

// @ivogabe